elm.json

```diff
@@ -25,7 +25,7 @@
             "Css.Global"
         ]
     },
-    "elm-version": "0.19.0 <= v < 0.20.0",
+    "elm-version": "0.19.1 <= v < 0.20.0",
```

Comment:
Is 0.19.1 required for this to work? (If not, I'd rather keep it compatible with 0.19.0!)

Reply:
Not required. Back to 0.19.0